Laravel.md

```diff
@@ -0,0 +1,258 @@
+# Laravel with Homestead : Web app Development and Management
+
+[PHP Docs](http://php.net/manual/en/)
+
+[Laravel Docs](https://laravel.com/docs/5.5)
+
+
+## What is Homestead
+
+- Homestead is a pre-packaged vagrant box that provides you will all the development tools that you need by default
+
+- Once you have setup your virtual machine and vagrant, you simply add homestead box to it and all of the development tools are installed automatically for you. This saves you the time of individually installing the required software.
+
+- Laravel homestead isn’t limited to the Laravel framework.
+
+- You can generally development any PHP web application using homestead.
+
+
+## Advantages of Homestead
+
+- Local development environment simulates production environment
+
+- Needed software out of the box
+
+- VM’s can be disposable and recreated within minutes without affecting application source code
+
+- Easily share your vagrantfile with others
+
+
+## Introduction to PHP and Laravel
+
+PHP, very much like node.js or ruby is a back-end language. It is especially suited for web development.
+
+Laravel is a front-end framework for PHP. 
+
+Writing PHP and Laravel feels very similar to writing Ruby and Rails respectively. 
+
+
+An example of this can be seen here when we make controllers for websites:
+
+
+### Rails
+```
+class ForumsController < ApplicationController
+
+  def index
+    @forums = Forum.all
+  end
+
+end
+```
+
+### Laravel
+```
+<?php
+
+class FarmController extends Controller{
+  public function index(){
+    $farms = Farm::all();
+    return view('farms.index', ['farms' => $farms]);
+  }
+}
+
+?>
+```
+
+## How to Setup and Install
+
+- First we needs to install virtual box and vagrant:
+
+  [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+
+  and
+
+  [Vagrant](https://www.vagrantup.com/downloads.html)
+
+
+- Once these two are installed type the following in your terminal:
+
+```
+  vagrant box add laravel/homestead
```

Comment:
Potentially consider left-justifying this section, similar to the `git clone` line (line no. 92)